engines/griffon/dialogs.cpp

```diff
@@ -267,14 +267,14 @@ void GriffonEngine::configMenu() {
 		rcDest.setWidth(320);
 		rcDest.setHeight(240);
 
-		_cloudImg->blit(*_videoBuffer, 0, 0, Graphics::FLIP_NONE, &rcDest);
+		_cloudImg->blit(*_videoBuffer, 0, 0, Graphics::FLIP_NONE, &rcDest,TS_ARGB(128,255,255,255));
```

Comment:
The new code piece does not conform to our formatting guidelines: there must be spaces after commas.